src/user_input_compiler.rs

```diff
@@ -794,57 +794,33 @@ mod test {
   
   #[test]
   fn webcam() {
-    match Input::from( format!("/webcam") ) {
-      WebCam{ nth } => if let None = nth { assert!(true) } else { assert!(false) },
-      _ => assert!(false),
-    }
+    assert_eq!( WebCam{ nth: None }, Input::from( format!("/webcam") ) )
   }
   
   #[test]
   fn webcam_negative_1() {
-    let r = Input::from( format!("/webcam -1") );
-    
-    if let InvalidSyntax(_) = r {
-    } else {
-        println!("Unexpected: {:?}", r);
-        assert!(false) ;
-    }
+    assert_eq!( InvalidSyntax("Expected positive Integer".into()), Input::from( format!("/webcam -1") ))
   }
   
   #[test]
   fn webcam_negative_13() {
-    let r = Input::from( format!("/webcam -13") );
-    
-    if let InvalidSyntax(_) = r {
-    } else {
-        println!("Unexpected: {:?}", r);
-        assert!(false) ;
-    }
+    assert_eq!( InvalidSyntax("Expected positive Integer".into()), Input::from( format!("/webcam -13") ))
```

Comment:
Hmm... The test for the exact error message is maybe a bit too much, but I would leave it for now until it gets annoying to fix the unit tests.